### PR TITLE
README: add doc about easy etcd cluster bootstrap

### PR DESCRIPTION
**Pull request by yichengq**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,10 +55,24 @@ curl -L http://127.0.0.1:4001/v2/keys/mykey -XPUT -d value="this is awesome"
...
 
-You have successfully started an etcd on a single machine and written a key to the store. Now it's time to dig into the full etcd API and other guides.
+You have successfully started an etcd and written a key to the store.
+
+### Running local etcd cluster
+
+First install [goreman](https://github.com/mattn/goreman), then start a three-member cluster of etcd:
+
+```sh
+goreman start
```
Comment (robszumski) [inline README.md]: What does this get you?

Comment (xiang90) [inline README.md]: probably we can describe like this:

`goreman` will run the Procfile script inside /etcd. 
The script contains an local example 3-member cluster and a proxy:
```
etcd1: bin/etcd -name infra1 -listen-client-urls http://localhost:4001 -advertise-client-urls http://localhost:4001 -listen-peer-urls http://localhost:7001 -initial-advertise-peer-urls http://localhost:7001 -initial-cluster-token etcd-cluster-1 -initial-cluster 'infra1=http://localhost:7001,infra2=http://localhost:7002,infra3=http://localhost:7003' -initial-cluster-state new
etcd2: bin/etcd -name infra2 -listen-client-urls http://localhost:4002 -advertise-client-urls http://localhost:4002 -listen-peer-urls http://localhost:7002 -initial-advertise-peer-urls http://localhost:7002 -initial-cluster-token etcd-cluster-1 -initial-cluster 'infra1=http://localhost:7001,infra2=http://localhost:7002,infra3=http://localhost:7003' -initial-cluster-state new
etcd3: bin/etcd -name infra3 -listen-client-urls http://localhost:4003 -advertise-client-urls http://localhost:4003 -listen-peer-urls http://localhost:7003 -initial-advertise-peer-urls http://localhost:7003 -initial-cluster-token etcd-cluster-1 -initial-cluster 'infra1=http://localhost:7001,infra2=http://localhost:7002,infra3=http://localhost:7003' -initial-cluster-state new
proxy: bin/etcd -proxy=on -bind-addr 127.0.0.1:8080 -initial-cluster 'infra1=http://localhost:7001,infra2=http://localhost:7002,infra3=http://localhost:7003'
```

File: README.md [hunk 2]
```diff
@@ -55,10 +55,26 @@ curl -L http://127.0.0.1:4001/v2/keys/mykey -XPUT -d value="this is awesome"
...
 ```
 
-You have successfully started an etcd on a single machine and written a key to the store. Now it's time to dig into the full etcd API and other guides.
+You have successfully started an etcd and written a key to the store.
+
+### Running local etcd cluster
+
+First install [goreman](https://github.com/mattn/goreman), which manages Procfile-based applications.
+
+We write [a Procfile script](./Procfile) which contains an local example cluster, and you can start it:
```
Comment (robszumski) [inline README.md]: What about:

Our [Profile script]() will set up a local example cluster. You can start it with:

Comment (yichengq) [inline README.md]: will do

Comment (yichengq) [inline README.md]: done

**Comment by robszumski** [comment]
LGTM

**Pull request merged**